runtime/services/catalog/migrator/sources/sources.go

```diff
@@ -361,6 +362,13 @@ func connectorVariables(src *runtimev1.Source, env map[string]string, repoRoot s
 		vars["dsn"] = repoRoot
 	case "bigquery":
 		vars["google_application_credentials"] = env["google_application_credentials"]
+	case "postgres":
+		props := src.Properties.AsMap()
+		if dsn, ok := props["pg_database_url"]; ok {
+			vars["dsn"] = dsn
+		} else if allowHostAccess, _ := vars["allow_host_access"].(bool); allowHostAccess {
+			vars["dsn"] = os.Getenv("PG_DATABASE_URL")
+		}
```

Comment:
1. Can we remove this and instead rely on only scoped variables (`connector.name.database_url`) – we don't have backwards compatibility to worry about here
2. Instead of `"dsn"`, can we refactor the driver to consume `database_url` instead (for consistency to avoid hard-coding property name mappings between code files and internals)